elastic/e2e-testing
# Build binaries via goreleaser

This handles building for all supported architectures including running packr
for embedding the binary files.

This allows us to easily extend our release process for tagging official cli
releases, building in various package formats and publishing to different
package registries

Signed-off-by: Adam Stokes <51892+adam-stokes@users.noreply.github.com>



## What does this PR do?



Utilizes a release tool for easily building binaries for all arches and publishing

## Why is it important?



Simplifies the process in building binaries for this project

## Checklist




## How to test this PR locally



Make sure `goreleaser` is installed and you've set your go environment to the version you want to build the binaries against:

```
$ brew install goreleaser/tap/goreleaser
$ eval $(gvm 1.14.12)
$ make build
```

Verify the distributions are built and placed in their respective arch directories under `dist/`
